docs/build/smart-contracts/getting-started/storing-data.mdx

```diff
@@ -27,12 +27,12 @@ This tutorial assumes that you've already completed the previous steps in Gettin
 
 ## Adding the increment contract
 
-The `stellar contract init` command allows us to initialize a new project with any of the example contracts from the [soroban-examples](https://github.com/stellar/soroban-examples) repo, using the `--with-example` (or `-w`) flag.
+The `stellar contract init` command allows us to initialize a new project with any of the example contracts from the [soroban-examples](https://github.com/stellar/soroban-examples) repo, using the `--name` flag, to specify a new contract's name.
 
-It will not overwrite existing files, so we can also use this command to add a new contract to an existing project. Run the command again with a `--with-example` flag to add an `increment` contract to our project. From inside our `soroban-hello-world` directory, run:
+It will not overwrite existing files unless we explicily pass in `--ovewrite` flag. Run the command with a `--name` flag to add an `increment` contract to our project. From inside our `soroban-hello-world` directory, run:
 
 ```sh
-stellar contract init ./ --with-example increment
+stellar contract init . --name increment
```

Comment:
At the moment this command copies the contents of the `hello-world` contract from the binary, and renames the directory as `increment`. 

I think that the only way to get the contracts from [soroban-examples](https://github.com/stellar/soroban-examples) we still need to use `--with-example` for now.

Reply:
Another idea would be to use `stellar contract init . --name increment` still and then copy the increment lib.rs and test.rs files over into the new files that are created. but that may be confusing. 🤔

Reply:
Oh wow, that's the behavior? Hmm, okay I will document as it currently works, but the name seems misleading.

Reply:
Would appreciate another look next week 🙏

Reply:
I agree, the name is a bit confusing. I think that the intention of that flag was to allow a user to add a new contract with the name of their choosing - but it uses the contract code from Hello World as default code.